doc/rfc/0000-rfc-repository.md

```diff
@@ -0,0 +1,168 @@
+open source contributors and should add little overhead.
+
+## Motivation
+
+RFCs in OpenTitan are used for decisions that need consensus from the technical
+committee. A summary of the RFC process as documented [here][rfc-process]:
+
+1. Write up your RFC somewhere.
+2. Create an issue in the OpenTitan repo with a link to your RFC.
+3. Label the issue with `RFC:Proposal`, then `RFC:TC review` when ready.
+4. Pending RFCs will be discussed in the next technical committee meeting.
+5. Approved RFCs have their label changed to `RFC:Approved`.
+6. Rejected RFCs have their label removed.
+
+Typically, authors write their RFCs in one of two places:
+
+* A Google Doc, accessible only to select people.
+* The GitHub issue itself as Markdown.
+
+RFCs stored like this are not discoverable and not portable. In the case
+of Google Docs, they are not accessible to everybody, even within the
+opentitan.org domain. RFCs are reviewed in private by the technical committee
+and project director.
+
+This makes it difficult (or impossible) for contributors to find documentation
+of what decisions have been made and why. Adopting more open processes for
+project governance will help the OpenTitan project be an exemplary case for
+open hardware development.
+
+## Explanation
+
+RFCs can be very important documentation artefacts. They show exactly what
+decisions have been made and the factors used to come to those decisions. As an
+open source project, we can make this process and the aftefacts more open. This
+RFC proposes applying our open source development model to the RFC process.
+
+We would do this by having RFCs:
+
+* Written as Markdown text documents.
+* Stored and tracked in a Git repo.
+* Proposed through GitHub PRs.
+* Reviewed and accepted/rejected through GitHub.
+
+The workflow for proposing and reviewing would then be the same as contributing
+any code or documentation to OpenTitan.
+
+### RFC file structure
+
+Having a repo of RFCs like this brings them closer to any other kind of
+documentation. To make best use of RFCs as documentation, this RFC also
+proposes adding some structure to RFCs:
+
+* RFCs are dated (and authored if necessary).
+* RFCs are based on some template file to help the author fully consider all
+  aspects of their proposal.
+* RFCs are assigned numeric IDs when accepted.
+* RFC numbers should start at 0001 and increase monotonically with date
+  accepted.
+* There should be no gaps in RFC numbers.
+
+Having RFCs as plain documentation lets us reference them when, for example:
+
+* Justifying the need for an issue or PR in OpenTitan.
+* Explaining why something was designed as it was in documentation.
+* Pointing to similar or conflicting RFCs to new proposals.
+
+### Private RFCs
+
+How and where private RFCs are stored is out of scope for this RFC, except to
+say that any redacted form of a private RFC that can be included must be. At the
+very least, private RFCs are still assigned the next RFC number when accepted.
```

Comment:
This implies that we have private RFCs. AFAIK that was never the intention. While there may be certification-sensitive information when RFCs are submitted to the TC, I think even the current process intends for the final proposal to be made public (and ideally translated to Markdown, if originally submitted as a GDoc).

Reply:
I just assumed the private Google Docs contained sensitive information.

I've removed this section.